examples/tracing-http-propagator/src/client.rs

```diff
@@ -37,21 +58,22 @@ async fn send_request(
     global::get_text_map_propagator(|propagator| {
         propagator.inject_context(&cx, &mut HeaderInjector(req.headers_mut().unwrap()))
     });
+    req.headers_mut()
+        .unwrap()
+        .insert("baggage", "is_synthetic=true".parse().unwrap());
```

Comment:
hardcoded instead of putting it to context and then letting the propagator do this job. In future, we can modify this.